### ISSUE TITLE
room-join event not trigger

### ISSUE THREAD

**Issue by kis87988** [lijiarui/wechaty-puppet-padchat#133]
### Run `npm run doctor` or `wechaty run doctor`(for docker user), paste output here 
docker run in container
wechaty-puppet-padchat@0.9.11

### Expected behavior
```typescript
bot
  .on('room-join', async function (room, inviteeList, inviter)
  {
    const room_topic = await room.topic()
    const invitee_list = inviteeList.map(c => c.name()).join(',')
    const inviter_name = inviter.name() 
    const invitee_alias = await inviteeList[0].alias()
    
    log.info('Bot', 'EVENT: room-join - Room "%s" got new member "%s", invited by "%s"',
      room_topic,
      invitee_list,
      inviter_name,
    )
  }
```

### Actual behavior
room-invite event not trigger 

### Full Output Logs

```shell
$ docker create -t -i --name=$docker_name \
--mount type=bind,source="${dir}",target=/bot \
-e WECHATY_LOG=silly \
-e TZ="America/Los_Angeles" \
zixia/wechaty:latest mybot.ts
$ docker start -a $docker_name 
```

<details>
<summary>
Show Logs
</summary>

### Paste the full output logs here with `WECHATY_LOG=silly` set 

```shell
# PASTE FULL LOG OUTPUT AT HERE:
17:16:19 SILL PuppetPadchat onPadchatMessageRoomEventJoin() roomJoinEvent="{"inviteeNameList":["爱吃玫瑰姐"],"inviterName":"BAILEY","roomId":"6222123866@chatroom"}"
17:16:19 VERB PuppetPadchat onPadchatMessageRoomEventLeave({id=165975270185661485})
17:16:19 VERB PuppetPadchat onPadchatMessageRoomEventTopic({id=165975270185661485})
17:16:19 VERB PuppetPadchat onPadchatMessageRoomEvent({id=1}) roomJoin retry(attempt=0)
17:16:19 VERB PuppetPadchat roomMemberList(6222123866@chatroom)
17:16:19 VERB PuppetPadchatManager getRoomMemberIdList(6222123866)
17:16:19 VERB PuppetPadchatManager getRoomMemberIdList(6222123866) length=102
17:16:19 SILL PuppetPadchat roomMemberList()=102
```
after 10mins log but still not trigger
</details>

**Comment by kis87988** [comment]
This may fix after wechaty-puppet-padchat@0.9.16. 
Still testing.

**Comment by kis87988** [comment]
This has been fix after 0.9.16 refer to commit https://github.com/lijiarui/wechaty-puppet-padchat/commit/fe21995518bf6cf144e7954ea8643baa44b48742
